inference-engine/src/transformations/src/transformations/convert_opset1_to_legacy/convert_nms_5_to_legacy.cpp

```diff
@@ -0,0 +1,123 @@
+// Copyright (C) 2018-2020 Intel Corporation
```

Comment:
Remove all transformations from this PR. We need just nGraph operation with unit tests.

Reply:
Done.